### PR TITLE
More mechanical cleanup of tables extension

### PR DESCRIPTION
**Pull request by pakrym**
Namespaces, hide some APIs, move files around, reformat, etc.

### CODE REVIEW

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/AzureTablesWebJobsStartup.cs [hunk 1]
```diff
@@ -5,15 +5,15 @@
 using Microsoft.Azure.WebJobs.Hosting;
 using Microsoft.Extensions.Hosting;
 
-[assembly: WebJobsStartup(typeof(AzureStorageWebJobsStartup))]
+[assembly: WebJobsStartup(typeof(AzureTablesWebJobsStartup))]
 
 namespace Microsoft.Azure.WebJobs.Extensions.Storage
```
Comment (JoshLove-msft) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/AzureTablesWebJobsStartup.cs:10]:
```suggestion
namespace Microsoft.Azure.WebJobs.Extensions.Tables
```

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/Microsoft.Azure.WebJobs.Extensions.Tables.csproj [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-    <Version>1.0.0-beta.1</Version>
-    <Description>This package extends the Microsoft.Azure.WebJobs library with Table triggers using Microsoft Azure Storage.</Description>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <AssemblyName>Microsoft.Azure.WebJobs.Extensions.Table</AssemblyName>
-    <RootNamespace>Microsoft.Azure.WebJobs.Extensions.Table</RootNamespace>
-    <NoWarn>$(NoWarn);CS1606;CS0618;AZC0001;CS1591;SA1636;SA1028;CA1200</NoWarn>
-  </PropertyGroup>  
+    <PropertyGroup>
+        <Version>1.0.0-beta.1</Version>
+        <Description>This package extends the Microsoft.Azure.WebJobs library with Table triggers using Microsoft Azure Storage.</Description>
```
Comment (JoshLove-msft) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/Microsoft.Azure.WebJobs.Extensions.Tables.csproj:5]: And cosmos?

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/ParameterizedTableEntityPath.cs [hunk 3]
```diff
@@ -1,51 +1,42 @@
...
         private readonly BindingTemplate _tableNameTemplate;
         private readonly BindingTemplate _partitionKeyTemplate;
         private readonly BindingTemplate _rowKeyTemplate;
+
         public ParameterizedTableEntityPath(BindingTemplate tableNameTemplate, BindingTemplate partitionKeyTemplate,
             BindingTemplate rowKeyTemplate)
         {
             Debug.Assert(tableNameTemplate.HasParameters || partitionKeyTemplate.HasParameters
-                || rowKeyTemplate.HasParameters);
+                                                         || rowKeyTemplate.HasParameters);
```
Comment (JoshLove-msft) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/ParameterizedTableEntityPath.cs:21]: nit: indent should be removed

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableExtensions.cs [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 using System;
-using System.Collections.Generic;
 using System.Globalization;
-using System.Text;
 using Microsoft.Azure.Cosmos.Table;
-namespace Microsoft.Azure.WebJobs.Host.Tables
+
+namespace Microsoft.Azure.WebJobs.Extensions.Tables
 {
     internal static class TableExtensions
```
Comment (JoshLove-msft) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableExtensions.cs:10]: possibly combine with other extensions class

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/Microsoft.Azure.WebJobs.Extensions.Tables.csproj (hunk 2 above)
Comment (christothes) [reply sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/Microsoft.Azure.WebJobs.Extensions.Tables.csproj:5]: Probably just say Azure Tables service.

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/api/Microsoft.Azure.WebJobs.Extensions.Tables.netstandard2.0.cs [hunk 5]
```diff
@@ -42,18 +22,6 @@ public partial class TableAttribute : System.Attribute, Microsoft.Azure.WebJobs.
         public int Take { get { throw null; } set { } }
```
Comment (christothes) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/api/Microsoft.Azure.WebJobs.Extensions.Tables.netstandard2.0.cs:22]:
```suggestion
        public int MaxPerPage { get { throw null; } set { } }
```

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttribute.cs [hunk 6]
```diff
@@ -66,43 +72,34 @@ public TableAttribute(string tableName, string partitionKey, string rowKey)
...
+        public string Filter { get; set; }
+
         /// <summary>
         /// Used with filter. RowKey should be null.
         /// </summary>
-        public int Take
-        {
-            get; set;
-        }
+        public int Take { get; set; }
```
Comment (christothes) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttribute.cs:101]:
```suggestion
        public int MaxPerPage { get; set; }
```

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableEntityPath.cs [hunk 7]
```diff
@@ -1,58 +1,61 @@
...
         }
+
         public static TableEntityPath ParseAndValidate(string value)
         {
             TableEntityPath path;
             if (!TryParseAndValidate(value, out path))
             {
                 throw new InvalidOperationException("Table entity identifiers must be in the format " +
-                    "TableName/PartitionKey/RowKey and must meet table naming requirements.");
+                                                    "TableName/PartitionKey/RowKey and must meet table naming requirements.");
```
Comment (christothes) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableEntityPath.cs:38]: combine to a single line?

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/api/Microsoft.Azure.WebJobs.Extensions.Tables.netstandard2.0.cs (hunk 5 above)
Comment (pakrym) [reply sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/api/Microsoft.Azure.WebJobs.Extensions.Tables.netstandard2.0.cs:22]: We are keeping attributes as compatible with the previous version as possible.

**Pull request merged**